src/modules/Map/helpers/AbstractMapControl.js

```diff
@@ -7,9 +7,10 @@ export class AbstractMapControl extends React.Component {
   static Container = 'div';
 
   onAdd (map) {
+    const { disabled } = this.props;
     this.map = map;
     this.container = document.createElement('div');
-    this.container.className = this.constructor.containerClassName;
+    this.container.className = `${this.constructor.containerClassName}${disabled ? ' mapboxgl-ctrl--disabled' : ''}`;
```

Comment:
```suggestion
    this.container.className = this.constructor.containerClassName;
    if (disabled) {
      this.container.classList.add('mapboxgl-ctrl--disabled');
    }
```

```suggestion
    this.container.className = classnames({
      [this.constructor.containerClassName]: true,
      'mapboxgl-ctrl--disabled' : disabled,
    });
```

```suggestion
    this.container.className = classnames(
      this.constructor.containerClassName,
      { 'mapboxgl-ctrl--disabled' : disabled },
    );
```

```suggestion
    this.container.className = `${this.constructor.containerClassName} ${disabled ? 'mapboxgl-ctrl--disabled' : ''}`;
```

```suggestion
    this.container.className = `${this.constructor.containerClassName + (disabled ? ' mapboxgl-ctrl--disabled' : '')}`;
```

Ou si on se fiche de IE (ou qu'on ajoute des polyfills) :

```suggestion
    this.container.className = this.constructor.containerClassName;
    this.container.classList.toggle('mapboxgl-ctrl--disabled', disabled);
```